boards/st-stm32f303re-nucleo-dev-board/HwInit.cxx

```diff
@@ -66,15 +66,16 @@ static Stm32Uart uart0("/dev/ser0", USART2, USART2_IRQn);
 static Stm32Can can0("/dev/can0");
 
 /** EEPROM emulation driver. The file size might be made bigger. */
-static Stm32EEPROMEmulation eeprom0("/dev/eeprom", 4000);
+static Stm32EEPROMEmulation eeprom0("/dev/eeprom", 8192);
```

Comment:
let's take 10% off of this maximum. We have a problem when a file size hits the maximum and we will wear out the flash super fast, even though it will appear to work correctly.

Reply:
done in main.cxx